donate/templates/payment/card_master.html

```diff
@@ -0,0 +1,186 @@
+                                    <label class="form-item__label" for="cvv">{% trans "CVV" %}</label>
+                                    <div class="form-item__tooltip-wrapper">
+                                        <div class="form-item__dummy-placeholder payments__braintree-hosted-field" id="cvv"></div>
+                                    </div>
+                                </div>
+                            </div>
+
+                            <footer class="donate-form__authorization">
+                                {% if payment_frequency == 'monthly' %}
+                                    <p>{% blocktrans with email='donate@mozilla.org' %}I authorize Mozilla Foundation to automatically charge my card every month in the amount indicated above, starting today and continuing each month on the same day, until I cancel. I understand that I must cancel at least 5 days before the next scheduled donation by email to <a href='mailto:{{ email }}'>{{ email }}</a>, and that I may only request a refund within 15 days from the date of a donation.{% endblocktrans %}</p>
+                                {% else %}
+                                    <p>{% trans "I authorize Mozilla Foundation to charge my card in the amount indicated above. I understand that I may only request a refund within 15 days from the date of the donation." %}</p>
+                                {% endif %}
+                            </footer>
+
+                        </div>
+
+                        <div class="form__section form__section--actions form__section--bottom-margin">
+
+                            <button value="donate" class="button button--large button--full button--primary button--icon" id="payments__payment-submit" disabled>
+                                <svg class="button__icon" width="24" height="24">
+                                    <use xlink:href="#padlock"></use>
+                                </svg>
+                                <div class="button__label">
+                                    {% format_currency request.LANGUAGE_CODE currency_info.code form.amount.value as formatted_amount %}
+                                    {% if payment_frequency == 'monthly' %}
+                                        {% blocktrans with formatted_amount=formatted_amount span_class='button__value js-donation-value' %}Donate <span class="{{ span_class }}">{{ formatted_amount }}</span> per month{% endblocktrans %}
+                                    {% else %}
+                                        {% blocktrans with formatted_amount=formatted_amount span_class='button__value js-donation-value' %}Donate <span class="{{ span_class }}">{{ formatted_amount }}</span>{% endblocktrans %}
+                                    {% endif %}
+                                </div>
+                            </button>
+
+                        </div>
+
+                        {% block donate_footer %}
```

Comment:
This file is exactly what the `donate/templates/payment/card.html` was with a new block `donate_footer`